content/en/docs/spinnaker-user-guides/using-dinghy.md

```diff
@@ -15,7 +17,7 @@ Before you can use this feature, please ensure you have [configured]({{< ref "di
 
 ## How it works in a nutshell
 
-GitHub (or BitBucket) webhooks are sent off when you modify either the Templates or the Module definitions. The Dinghy service looks for and fetches all dependent modules and parses the template and updates the pipelines in Spinnaker. The pipelines get automatically updated whenever a module that is used by a pipeline is updated in VCS. This is done by maintaining a dependency graph.  Dinghy will look for a `dinghyfile` in all directories, not just the root path.  Unless otherwise configured, Dinghy will process changes found in the master branch. For more information on how to configure branches, see [Custom branch configuration]({{< ref "dinghy-enable#custom-branch-configuration" >}})
+GitHub (or BitBucket) webhooks are sent off when you modify either the Templates or the Module definitions. The Dinghy service looks for and fetches all dependent modules and parses the template and updates the pipelines in Spinnaker. The pipelines get automatically updated whenever a module that is used by a pipeline is updated in VCS. This is done by maintaining a dependency graph.  Dinghy will look for a `dinghyfile` in all directories, not just the root path.  Unless otherwise configured, Dinghy will process changes found in the main branch. For more information on how to configure branches, see [Custom branch configuration]({{< ref "dinghy-enable#custom-branch-configuration" >}})
```

Comment:
Unless otherwise configured, Dinghy will process changes found in the master branch.

I think it should stay "master" because the change hasn't happened, or say **Primary (by default, the "master" branch)**
Until we have the change created and pushed, we can't say it read the "main" branch otherwise it will cause confusion as people will expect it to be reading the branch named "main" which is what the actual issue is